assets/sass/application.sass

```diff
@@ -58,14 +58,48 @@ $path: "/assets/images/"
     width: 75%
```

Comment:
As part of this PR, Ive removed the duplication of sass files, opting for the indented version seeing as this is used as part of the build process. It was slightly out of date compared to the scss file though